src/js/contentscripts/socialwidgets.js

```diff
@@ -330,7 +330,7 @@ function replaceIndividualButton(tracker) {
 *                            replaced
 */
 function getTrackerData(callback) {
-  chrome.runtime.sendMessage({checkReplaceButton:document.location}, function(response) {
+  chrome.runtime.sendMessage({checkReplaceButton:document.location.hostname}, function(response) {
```

Comment:
If we are not using `checkReplaceButton`'s value back in `webrequest.js`, let's send `true` instead.